docs/src/tutorials/beginner_tutorial.md

```diff
@@ -0,0 +1,229 @@
+
+```@example beginner_tutorial
+my_variable = 42
+```
+
+For basic math operations, we can just use the following operators:
+
+```@example beginner_tutorial
+sum = 3 + 2
+product = 3 * 2
+difference = 3 - 2
+quotient = 3 / 2
+power = 3 ^ 2 
+modulus = 3 % 2
+```
+
+To define a function in Julia, we first write `function` followed by the function's name, its parameters, the commands and `end`:
+
+```@example beginner_tutorial
+function tutorial(parameter)
+    # Your code goes here
+    println("The input was $parameter.")
+end
+```
+
+And we can simply call it by its name:
+ 
+```@example beginner_tutorial
+tutorial("Neuroblox")
+#This should print "The input was Neuroblox."
+```
+
+## Plotting graphs
+
+We can plot graphs in Julia using the `Plots.jl` package. To add this package we use the terminal:
+
+```@example beginner_tutorial
+using Pkg
+Pkg.add("Plots")
+# This will use the package manager to explicitely add the Plots.jl package
+
+using Plots
+# This will load the package
+```
+
+Inside this package, there are several built-in functions. First, we have to tell Plots.jl which backend we will be using for rendering plots.
+
+In this tutorial we will be using `GR backend`, but there are several others and you can find them here: https://docs.juliaplots.org/latest/backends/
+
+To call GR, we simply call it as the parameterless function `gr()`. After that, we need an interval and a data set to generate a graph. In general, we'll have something like this:
+
+```@example beginner_tutorial
+using Plots
+
+# Set GR as the backend for Plots.jl
+gr()
+
+# Create some data (an interval and a random 10-element vector)
+x = 1:10
+y = rand(10)
+
+# Plot the data
+plot(x, y, label="Random Data", xlabel="X-axis", ylabel="Y-axis", title="Example")
+```
+
+You can try running the code above in your Julia terminal using your own data to see how it works.
+
+## Solving differential equations
+
+An essential package we'll be using is `DifferentialEquations.jl`, a library designed to solve various types of differential equations, such as ODEs, SDEs and PDEs.
+
+To use the `DifferentialEquations` library, we need to define a differential equation as a function, solve it and then plot it (we know how to do the last part).
+
+Another useful library we'll be using is `ModelingToolkit.jl`, a Julia package designed to make our lives easiear when dealing with complex models and performing tasks such as model simplification, parameter estimation and code generation.
+
+Let's solve the ODE $\frac{du}{dt}=-a \cdot u$  using this package. First we need to define what are our variables and what are our parameters. For this, we'll use the `@variables`and `@parameters` macros from `ModelingToolkit`:
+
+```@example beginner_tutorial
+# Install and import these packages
+using Pkg
+Pkg.add("ModelingToolkit")
+Pkg.add("DifferentialEquations")
+using ModelingToolkit
+using DifferentialEquations
+
+# Define the time variable
+@variables t
```

Comment:
It's okay to do this, but it might be a better idea to just use MTK's builtin `t` and `D`, i.e.
```
using ModelingToolkit: t_nounits as t, D_nounits as D
```
since that's the more recommended way of doing things nowadays, you'll have less clashing definitions of `t` and `D` floating around, but it's also not a big deal